realm/realm-library/build.gradle

```diff
@@ -6,6 +6,7 @@ apply plugin: 'com.github.dcendents.android-maven'
 apply plugin: 'maven'
 apply plugin: 'findbugs'
 apply plugin: 'checkstyle'
+apply plugin: 'jacoco'
```

Comment:
Is this needed? Last time I checked the source this import was done by the android plugin